src/main/java/Factorial/Factorial.java

```diff
@@ -0,0 +1,16 @@
+package Factorial;
+
+public class Factorial {
+    public static void main(String[] args) {
+        int number = 4;
+        System.out.println(findFactorial(number));
+    }
+
+    public static int findFactorial(int number) {
+        int i,factorialResult=1;
+        for (i =1; i <= number; i++) {

```

Comment:
for'un icinde tek satir kod varsa { kullanmana gerek yok. bana gore kod daha temiz duruyor.